tardis/montecarlo/enum.py

```diff
@@ -0,0 +1,58 @@
+from ctypes import c_int
+
+
+class EnumerationType(type(c_int)):
```

Comment:
Normally, when we copy code from the internet we like to attribute it with the original source. In science this is an important part to understand the source of information. Can you add one?